### DISCUSSION TITLE
Incorrect Port Mapping Documentation for Docker Deployment

### DISCUSSION THREAD

**Discussion by sergeyvslv** [0xJacky/nginx-ui#278] [General]
Hey!
I believe I noticed a small hiccup in the Docker deployment documentation. The example command seems to have a little mix-up in port mappings:
![photo_2024-02-08_15-36-19](https://github.com/0xJacky/nginx-ui/assets/87233585/4698677f-3466-4870-a396-67ee70f21b3e)
In the given example, the ports 8080 and 8443 of the host are bound to ports 80 and 443 of the container, not vice versa as indicated in the documentation.

**Comment by 0xJacky** [comment]
Thanks for your report, this issue has been fixed in c662505
